pythonManual/antora/antora.yml

```diff
@@ -1,14 +1,14 @@
 name: python-manual
 title: Neo4j Python Driver Manual
-version: '4.2'
+version: '4.3'
 start_page: ROOT:index.adoc
 nav:
 - modules/ROOT/content-nav.adoc
 asciidoc:
   attributes:
-    neo4j-version: '4.2'
-    neo4j-version-exact: '4.2.3'
-    neo4j-buildnumber: '4.2'
-    driver-version: "4.2"
-    python-driver-version: "4.2.1"
-    python-examples: "partial$driver-sources/python-driver/4.2.1/tests/integration/examples"
+    neo4j-version: '4.3'
+    neo4j-version-exact: '4.3.0'
+    neo4j-buildnumber: '4.3'
+    driver-version: "4.3"
+    python-driver-version: "4.3.0"
```

Comment:
4.3.1 is out now

Reply:
Thanks - I've updated to 4.3.1 in the Python file